build/kickstarts/base.ks.erb

```diff
@@ -20,6 +20,7 @@ repo --name=SCL --baseurl=http://mirror.centos.org/centos/6/SCL/x86_64/
 repo --name=manageiq-scl --baseurl=http://copr-be.cloud.fedoraproject.org/results/jrafanie/manageiq-scl/epel-6-x86_64/
 repo --name=ipa --baseurl=http://copr-be.cloud.fedoraproject.org/results/adelton/identity_demo/epel-6-x86_64/
 repo --name=certmonger --baseurl=http://copr-be.cloud.fedoraproject.org/results/nalin/certmonger/epel-6-x86_64/
+repo --name=ipa_scl --baseurl=http://copr-be.cloud.fedoraproject.org/results/adelton/identity_demo_scl/epel-6-x86_64/
```

Comment:
Good. can we have a one-off appliance built without the module mod_lookup_identity-selinux specified and only the ipa_scl repo ?  We'll need to test IPA with SELinux on that bad boy. "supposed" to work.